src/core/DOMEval.js

```diff
@@ -3,12 +3,22 @@ define( [
 ], function( document ) {
 	"use strict";
 
-	function DOMEval( code, doc ) {
+	function DOMEval( code, doc, node ) {
 		doc = doc || document;
 
 		var script = doc.createElement( "script" );
-
-		script.text = code;
+		if ( code ) {
+			script.text = code;
+		}
+		jQuery.each( [
+			"type",
+			"nomodule",
```

Comment:
`jQuery.each` is almost certainly overkill for what should be `for ( property in scriptProperties )` iteration when `node` is provided, and "nomodule" doesn't belong in the list because it is [ignored](https://html.spec.whatwg.org/#attr-script-nomodule) on module scripts (which are the only cases where we have a node here).